src/generic/grideditors.cpp

```diff
@@ -228,10 +228,17 @@ void wxGridCellEditorEvtHandler::OnChar(wxKeyEvent& event)
 // wxGridCellEditor
 // ----------------------------------------------------------------------------
 
-wxGridCellEditor::wxGridCellEditor()
+wxGridCellEditor::wxGridCellEditor(const wxGridCellEditor& other)
+    : wxGridCellWorker(other),
+      m_control(other.m_control)
 {
-    m_control = NULL;
-    m_attr = NULL;
+    if ( other.m_attr != NULL )
+    {
+        m_attr = other.m_attr->Clone();
+    }
```

Comment:
This leaves `m_attr` uninitialized if the other one is null, we should do
```suggestion
    m_attr = other.m_attr ? other.m_attr->Clone() : NULL;
```
instead.